Add ARIA Code Hints

Coder206 opened the pull request:
With the aria-atomic, aria-activedescendant and aria-flowto. Has been compared to WYSIWYG for ARIA, its a match. @RaymondLim Plus added "none" to aria-autocomplete

ingorichter commented:
Great addition. Thank you. Would you mind changing the formatting back to the previous style? BTW: did you sign the contributor license agreement (CLA)?

Coder206 commented:
@ingorichter I am not sure what you mean by "previous style". Yes I think I did!

ingorichter commented:
Looking at the diff, you can see that the previous formatting was different. Every entry was in one line.

Coder206 commented:
@ingorichter Ok!

Coder206 commented:
@ingorichter I can't tell if this is an error on my part. I don't think changing a format can fail a build...

Coder206 commented:
@ingorichter Is my work done here?

twang2218 commented:
@Coder206 The code style are different. Many irrelevant code have been touched in the first commit. Here is the screenshot.

<img width="1157" alt="2016-07-22 23 03 50" src="https://cloud.githubusercontent.com/assets/6299096/17057909/ac413d14-5060-11e6-9e2b-5b2edaf62197.png">

Did you turn on the code beautify/format? Maybe you can turn them off and restore the original format, and try commit again. Thanks.

Coder206 commented on src/extensions/default/HTMLCodeHints/HtmlAttributes.json:
```diff
@@ -1,227 +1,261 @@
-                                             "zu"], 
-                            "global": "true" },
-    "role":               { "attribOption": ["alert", "alertdialog", "article", "application", "banner", "button", "checkbox", "columnheader", "combobox", 
+    "accesskey":                { "attribOption": [], "global": "true" },
```
@twang2218 I thought I did... I am not sure what is going on.

twang2218 commented:
@Coder206 I send a pull request in your repo to you to fix the format issue some days ago, could you merge it, so this issue can be fixed?

Coder206 commented:
@twang2218 Ok sounds good.

ficristo commented:
@twang2218 you need to sign the CLA http://dev.brackets.io/brackets-contributor-license-agreement.html
While indirectly, you still have modified the source.

@ingorichter do you have time to review this?

twang2218 commented:
@ficristo ok. It's done.

Coder206 commented:
@ficristo I am quite sure I have filled it out. What makes you think I didn't?

ficristo commented:
@Coder206 Could you solve the merge conflicts?

Coder206 commented:
@ficristo I am very confused as to what I have to do know. I have a enabled edits from maintainers.

ficristo commented:
On your local branch you should merge master in it, solve eventual conflicts and push again your changes.

Coder206 commented:
@ficristo Ok I am wondering why this PR wasn't merged beforehand.

Coder206 commented:
Ping!

ficristo commented:
If your ping is about this not be merged, keep in mind that at the moment Adobe is focused on Dreamweaver so there is much help there.

Personally I was thinking @ingorichter would have reviewed this.
Then as some weeks ago, I was hoping a 1.8 release would come out, and I was waiting it before reviewing your PR myself.

For the conflicts they happen: you should resolve them.

Coder206 committed:
Add ARIA Code Hints

The purpose of the code is to add the ARIA code hints for HTML. Before
this, there were no code hints for the ARIA spec. This change allows
developers to get the ARIA codes while programming.

Coder206 commented:
@ficristo Done!

zaggino reviewed (changes requested):
You need to fix the diff, you merged master incorrectly removing `autocomplete` options: 
![image](https://cloud.githubusercontent.com/assets/1067319/19024541/650a57e4-8951-11e6-825a-cda36be947c1.png)

Coder206 commented:
Changes made :)

zaggino commented:
:+1: thanks @Coder206

ingorichter commented:
Sorry. I somehow missed that
